cmd/tsbs_load_clickhouse/creator.go

```diff
@@ -229,7 +230,7 @@ func createMetricsTable(db *sqlx.DB, tableSpec []string) {
 	}
 
 	sql := fmt.Sprintf(`
-			CREATE TABLE %s (
+			CREATE TABLE %s.%s (
 				created_date    Date     DEFAULT today(),
 				created_at      DateTime DEFAULT now(),
```

Comment:
created_at      DateTime DEFAULT now() CODEC(DoubleDelta, LZ4),

Reply:
Yeah, maybe created_date can modify to 
```
created_date    Date     DEFAULT today() CODEC(DoubleDelta, LZ4)
```

Is it right?

Reply:
>Yeah, maybe created_date can also modify to
> Is it right

No need. created_date is already well compressed. It's only several 100KB, because this columns has low cardinality.

Reply:
Ooooh I got it and I have foreced push it.

https://github.com/timescale/tsbs/commit/b05b72b61f80ea71d7eae9292a55b9e11d430505